sdk/servicebus/azure-servicebus/tests/async_tests/test_sessions_async.py

```diff
@@ -1167,4 +1167,29 @@ async def test_async_session_non_session_send_to_session_queue_should_fail(self,
             async with sb_client.get_queue_sender(servicebus_queue.name) as sender:
                 with pytest.raises(ServiceBusError):
                     message = ServiceBusMessage("Handler message")
-                    await sender.send_messages(message)
\ No newline at end of file
+                    await sender.send_messages(message)
+
+    @pytest.mark.asyncio
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedServiceBusResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', requires_session=True)
+    @pytest.mark.parametrize("uamqp_transport", uamqp_transport_params, ids=uamqp_transport_ids)
+    @ArgPasserAsync()
+    async def test_async_next_available_session_timeout_value(self, uamqp_transport, *, servicebus_namespace_connection_string=None, servicebus_queue=None, **kwargs):
+        if uamqp_transport==True:
+            pytest.skip("This test is for pyamqp only")
+        async with ServiceBusClient.from_connection_string(
+            servicebus_namespace_connection_string, logging_enable=False, uamqp_transport=uamqp_transport) as sb_client:
+            
+            receiver = sb_client.get_queue_receiver(servicebus_queue.name, session_id=NEXT_AVAILABLE_SESSION, max_wait_time=10)
+            start_time = time.time()
+            with pytest.raises(OperationTimeoutError):
+                await receiver.receive_messages(max_wait_time=5)
+            assert time.time() - start_time < 65 # Default service timeout value is 65 seconds
+            start_time2 = time.time()
+            with pytest.raises(OperationTimeoutError):
+                async for msg in receiver:
+                    pass
+            assert time.time() - start_time2 < 65 # Default service timeout value is 65 seconds
```

Comment:
How does this default service timeout interact with my client timeout value? Is that what would be used if I hadn't set a value in `get_queue_receiver` or `receive_messages`? Can I set a timeout value longer than 65 seconds?

Reply:
Yes so this service timeout is what would be the default if we didn't pass through the 'com.microsoft:timeout' value as link_properties. You can set it to be shorter or longer than 65 seconds, I tested up to 200 seconds and as little as 1 second and both worked.